pkg/utils/k8sutil.go

```diff
@@ -515,7 +515,17 @@ func EnsureFuncConfigMap(client kubernetes.Interface, funcObj *spec.Function, or
 
 	_, err = client.Core().ConfigMaps(funcObj.Metadata.Namespace).Create(configMap)
 	if err != nil && k8sErrors.IsAlreadyExists(err) {
-		_, err = client.Core().ConfigMaps(funcObj.Metadata.Namespace).Update(configMap)
+		// In case the ConfigMap already exists we should update
+		// just certain fields (to avoid race conditions)
+		var newConfigMap *v1.ConfigMap
+		newConfigMap, err = client.Core().ConfigMaps(funcObj.Metadata.Namespace).Get(funcObj.Metadata.Name, metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+		newConfigMap.ObjectMeta.Labels = funcObj.Metadata.Labels
+		newConfigMap.ObjectMeta.OwnerReferences = or
```

Comment:
or ?

Reply:
ok got it it is a function argument. I would not use a conditional key word as variable